packages/functional-tests/tests/signin/relyingParties.spec.ts

```diff
@@ -71,64 +76,75 @@ test.describe('severity-1 #smoke', () => {
 
   // https://testrail.stage.mozaws.net/index.php?/cases/view/1293362
   // https://testrail.stage.mozaws.net/index.php?/cases/view/1293469
-  test.fixme(
-    'can login to addons #1293362 #1293469',
-    async ({ credentials, pages: { page, login, settings } }, { project }) => {
-      test.skip(project.name !== 'production', 'uses prod addons site');
-
-      await page.goto('https://addons.mozilla.org/en-US/firefox/');
-      await page.getByRole('link', { name: 'Log in' }).click();
-      await page.waitForURL(/accounts\.firefox\.com/);
-      await login.login(credentials.email, credentials.password);
-      await page.waitForURL(
-        /addons\.mozilla\.org\/en-US\/firefox\/users\/edit/
-      );
-
-      // Can create AMO profile
-      await page
-        .getByRole('textbox', { name: /Display Name/ })
-        .fill('Example Name');
-      await page.getByRole('button', { name: 'Create my profile' }).click();
-
-      // User menu shows selected display name and user can log out of AMO
-      await page.waitForURL('https://addons.mozilla.org/en-US/firefox/');
-      await page.getByRole('button', { name: 'Example Name' }).hover();
-      await page.getByRole('button', { name: 'Log out' }).click();
-      await expect(page.getByText('You have been logged out.')).toBeVisible();
-
-      // Expect Mozilla Accounts to be signed in and AMO to be in Connected Services
-      await settings.goto();
-      await expect(page.getByRole('link', { name: 'Add-ons' })).toBeVisible();
-      await settings.avatarDropDownMenuToggle.click();
-      await settings.avatarMenuSignOut.click();
-    }
-  );
+  test('can login to addons #1293362 #1293469', async ({
+    target,
+    pages: { page, login, settings },
+    accountManager,
+  }, { project }) => {
+    test.fixme(true, 'Fix required as of 2024/02/09 (see FXA-9140).');
+    test.skip(project.name !== 'production', 'uses prod addons site');
```

Comment:
I might be confused here, but looks like we skip the test if we're running in production, but if it runs in stage, isn't this going to a hardcoded production AMO site on L89 and we're testing on production regardless of `project.name`?